pkg/sql/catalog/tabledesc/column.go

```diff
@@ -108,6 +108,20 @@ func (w column) GetDefaultExpr() string {
 	return *w.desc.DefaultExpr
 }
 
+// HasOnUpdate returns true iff the column has an on update expression set.
+func (w column) HasOnUpdate() bool {
+	return w.desc.HasOnUpdate()
+}
+
+// GetOnUpdateExpr returns the column on update expression if it exists,
+// empty string otherwise.
+func (w column) GetOnUpdateExpr() string {
+	if !w.HasOnUpdate() {
+		return ""
+	}
+	return *w.desc.OnUpdateExpr
+}
```

Comment:
This `OnUpdateExpr` field should be visited by `ForEachExprStringInTableDesc` in `tabledesc`.

Please add tests to cover a missing cross-reference in `validate.go` in the `tabledesc` package. By that I mean, a test case in which the `ON UPDATE` expression refers to a descriptor which doesn't exist, like an user-defined enum type value for example.